corelib-definitions/src/main/java/eu/europeana/corelib/definitions/edm/model/metainfo/ImageMetaInfo.java

```diff
@@ -0,0 +1,19 @@
+package eu.europeana.corelib.definitions.edm.model.metainfo;
+
+public interface ImageMetaInfo {
+    public Integer getWidth();
```

Comment:
Interfaces do not require the access level as they are always public. Remove public from all methods in this interface.